config/locales/en.yml

```diff
@@ -797,6 +797,9 @@ en:
         preamble: Provide in-depth information about how the server is operated, moderated, funded.
         rules_hint: There is a dedicated area for rules that your users are expected to adhere to.
         title: About
+      allow_referrer_origin:
+        desc: When your users click links to external sites, their browser may send the address of your Mastodon server as the referrer. Disable this if this would uniquely identify your users, e.g. if this is a personal Mastodon server.
```

Comment:
This looks ok, but things I'm worried are less uniquely identifying users, but having them inadvertently out something on them when clicking a link to an external service they are registered on (e.g. user on a LGBTQ instance logging in on a place where that could be an issue for them)